.github/workflows/publish-master-merges.yaml

```diff
@@ -0,0 +1,58 @@
+name: Publish all merges to master
+
+on:
+  push:
+    branches: ["master"]
+    paths:
+      - helm-chart/**
+      - scripts/**
+      - acceptance-tests/**
+      - .github/**
+  workflow_dispatch:
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions-ecosystem/action-get-latest-tag@v1
+        id: get-latest-tag
+      - id: commit
+        uses: pr-mpt/actions-commit-hash@v1
+      - uses: actions-ecosystem/action-bump-semver@v1
+        id: bump-semver
+        with:
+          current_version: ${{ steps.get-latest-tag.outputs.tag }}
+          level: prepatch
+      - id: set-version
+        run: echo "publish_version=${{ steps.bump-semver.outputs.new_version }}.${{steps.commit.outputs.short}}" >> $GITHUB_ENV
```

Comment:
you are a github actions search ninja 💪 Though I'm pretty sure something like 

```
echo "publish_version=${{ steps.bump-semver.outputs.new_version }}.$(${{ github.sha }} | cut -c 1-7)" >> $GITHUB_ENV
```

also works, like we do [here](https://github.com/SwissDataScienceCenter/renkulab-docker/blob/master/.github/workflows/build_and_push_to_docker.yml#L24) - this would reduce the number of dependencies

Reply:
done